source/NVDAObjects/UIA/wordDocument.py

```diff
@@ -215,10 +218,15 @@ def expand(self,unit):
 				self.setEndPoint(docInfo,"endToEnd")
 
 	def getTextWithFields(self,formatConfig=None):
-		if self.isCollapsed:
-			# #7652: We cannot fetch fields on collapsed ranges otherwise we end up with repeating controlFields in braille (such as list list list). 
-			return []
-		fields=super(WordDocumentTextInfo,self).getTextWithFields(formatConfig=formatConfig)
+		fields = None
+		if not self.isCollapsed:
+			rawText = self._rangeObj.GetText(2)

```

Comment:
Why getting 2 characters here instead of just one?

Reply:
Fetching 2 characters ensures that that the range does not deliberately span past the end of the cell and into the next cell and or out of the table. I've added a detailed code comment to the change now.